radix-engine-tests/src/common.rs

```diff
@@ -30,6 +30,10 @@ pub mod package_loader {
                 panic!("Package \"{}\" not found. Are you sure that this package is: a) in the blueprints folder, b) that this is the same as the package name in the Cargo.toml file?", name)
             }
         }
+
+        pub fn get_using_standard_compiler_profile(name: &str) -> (Vec<u8>, PackageDefinition) {
+            panic!("Package \"{}\" already compiled. Cannot use specific compiler profile in compile-blueprints-at-build-time mode.", name)
+        }
```

Comment:
Honestly, I'm fine if we just remove this method. This means the PackageLoader only support one compiler profile, within our engine tests.